cypress/integration/inventory/item-status-date-updates.spec.js

```diff
@@ -0,0 +1,362 @@
+    cy.log('###In transit###');
+
+    // check in item at service point assigned to its effective location
+    SwitchServicePoint.switchServicePoint(effectiveLocationServicePoint.name);
+    cy.visit(TopMenu.checkInPath);
+    CheckInActions.checkInItem(itemBarcode);
+    CheckInActions.openItemRecordInInventory(itemBarcode);
+    ItemVeiw.verifyUpdatedItemDate();
+    ItemVeiw.verifyItemStatus(ItemVeiw.itemStatuses.available);
+    cy.log('###Available###');
+
+    // create Page request on an item
+    cy.visit(TopMenu.requestsPath);
+    NewRequest.createNewRequest({
+      itemBarcode,
+      itemTitle: null,
+      requesterBarcode: userBarcode,
+      pickupServicePoint: effectiveLocationServicePoint.name,
+    });
+    openItem(instanceTitle, effectiveLocation.name, itemBarcode);
+    ItemVeiw.verifyUpdatedItemDate();
+    ItemVeiw.verifyItemStatus(ItemVeiw.itemStatuses.paged);
+    cy.log('###Paged###');
+
+    // check in item at a service point other than the pickup service point for the request
+    SwitchServicePoint.switchServicePoint(notEffectiveLocationServicePoint.name);
+    cy.visit(TopMenu.checkInPath);
+    CheckInActions.checkInItem(itemBarcode);
+    ConfirmItemInModal.confirmInTransitModal();
+    CheckInActions.openItemRecordInInventory(itemBarcode);
+    ItemVeiw.verifyUpdatedItemDate();
+    ItemVeiw.verifyItemStatus(ItemVeiw.itemStatuses.inTransit);
+    cy.log('###In transit###');
+
+    // check in item at the pickup service point for the page request
+    SwitchServicePoint.switchServicePoint(effectiveLocationServicePoint.name);
+    cy.visit(TopMenu.checkInPath);
+    CheckInActions.checkInItem(itemBarcode);
+    ConfirmItemInModal.confirmAvaitingPicupModal();
+    CheckInActions.openItemRecordInInventory(itemBarcode);
+    ItemVeiw.verifyUpdatedItemDate();
+    ItemVeiw.verifyItemStatus(ItemVeiw.itemStatuses.awaitingPickup);
+    cy.log('###Awaiting pickup###');
+
+    // check out item to user for whom page request was created
+    cy.visit(TopMenu.checkOutPath);
+    cy.wait(25000);
+    CheckOut.checkOutItem(userBarcode, itemBarcode);
+    ConfirmItemInModal.confirmAvaitingPicupCheckInModal();
+    CheckOut.openItemRecordInInventory(itemBarcode);
+    ItemVeiw.verifyUpdatedItemDate();
+    ItemVeiw.verifyItemStatus(ItemVeiw.itemStatuses.checkedOut);
+    cy.log('###Checked out###');
+
+    // declare item lost
+    cy.visit(TopMenu.usersPath);
+    UsersSearchPane.searchByKeywords(user.username);
+    UsersSearchPane.openUser(user.userId);
+    UsersCard.openLoans();
+    UsersCard.showOpenedLoans();
+    UserLoans.declareLoanLost();
+    ConfirmItemStatusModal.confirmItemStatus();
+    UserLoans.openItemRecordInInventory(itemBarcode);
+    ItemVeiw.verifyUpdatedItemDate();
+    ItemVeiw.verifyItemStatus(ItemVeiw.itemStatuses.declaredLost);
+    cy.log('###Declared lost###');
+
+    // renew item (through override)
+    cy.visit(TopMenu.usersPath);
```

Comment:
why do you need open usersPath several time?

Reply:
Done